### ISSUE TITLE
Support for Jetson Orin Nano requested

### ISSUE THREAD

**Issue by acostach** [balena-os/balena-jetson-orin#155]

**Comment by acostach** [comment]
@dremsol yes, this is the plan. We haven't started working on the Orin Nano yet.

**Comment by mishmashlabs** [comment]
Do you have a timeline for this?

**Comment by acostach** [comment]
@mishmashlabs short progress update, I've got the Nano running balenaOS from a NVME and will now look into rcm-booting it to load the flasher image from a USB key for provisioning.

The devkit I have is a 8G SD module and the fan spins at max speed (6000rpm as per the nvmfan conf) with both Ubuntu and balenaOS. It looks like it always reads the temp 0. Do you have a similar module, can you please tell me if yours behaves the same in Ubuntu?

**Comment by dremsol** [comment]
@acostach we received our Jetson Orin Nano DevKit from NVIDIA (sd-card). Would you like me to try?

**Comment by acostach** [comment]
@dremsol yes, if you can, please let me know.

**Comment by dremsol** [comment]
That probably requires a new machine to be defined as you are using a different one? `jetson-orin-nano-devkit-nvme` vs. `jetson-orin-nano-devkit`

**Comment by acostach** [comment]
@dremsol not sure what you are referring to exactly. Looking at the meta-tegra machines the only difference I spotted was the xml, the devkit booted from an sd-card and the devkit-nvme from a nvme. My intention is to provide a balenaOS flasher image which will provision the NVME, because the NVME offers more space and that's what most users asked for. But it's still possible to unpack the balenaOS image from the flasher, write it to an sd-card and boot it, for sure.

**Comment by dremsol** [comment]
@acostach for some reason I'm unable to download the image from [staging](dashboard.balena-staging.com), it returns a server error.

I've succeeded to build the image from `master`, and unwrapped the flasher image using [balena-image-flasher-unwrap](https://github.com/balena-os/balena-image-flasher-unwrap). After flashing the sd-card with BalenaEtcher my `Jetson Orin Nano Devkit` does not boot. I will pick up a cable to debug over serial console tomorrow.

**Comment by acostach** [comment]
Orin Nano Devkit (SD module) booting from NVME is now available in balena-cloud at v3.0.8+rev2
